addons/mail/static/tests/thread/thread.test.js

```diff
@@ -305,15 +298,10 @@ test("mark channel as fetched when a new message is loaded", async () => {
         asyncStep("rpc:channel_fetch");
     });
     setupChatHub({ opened: [channelId] });
+    const { waitForStoreFetch } = prepareStoreFetchListeners(["init_messaging", "discuss.channel"]);
     await start();
+    await waitForStoreFetch(["init_messaging", "discuss.channel"]);
     await contains(".o_menu_systray i[aria-label='Messages']");
-    await waitForSteps([
-        `lazy_session_info`,
-        `/mail/data - ${JSON.stringify({
-            fetch_params: [["discuss.channel", [channelId]]],
-            context: { lang: "en", tz: "taht", uid: serverState.userId, allowed_company_ids: [1] },
-        })}`,
-    ]);
```

Comment:
Note: highlight one of the many fixes of this PR:
fetch of `discuss.channel` (from chat hub) now going through `lazy_session_info` instead of making a separate RPC.